File: Week02/LeetCode-0217.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        // O(N^2) algorithm: Gets "Time Limit Exceeded" error
        for (size_t i=1; i<nums.size(); i++){
            for (int j=i-1; j>=0; j--){
                if (nums[j] == nums[i]) return true;
            } //end-for
        } // end-for

        return false;        
    } // end-containsDuplicate
};

File: Week02/LeetCode-0217.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        """
        O(N^2) algorithm
        """
        for i in range(len(nums)-1):
            for j in range(i+1, len(nums)):
                if nums[i] == nums[j]:
                    return True
        return False